crates/project/src/lsp_store.rs

```diff
@@ -4809,6 +4809,335 @@ impl LspStore {
         )
     }
 
+    pub fn definition(
```

Comment:
I really wanted to make things plural to reflect the actual behavior, but decided to make minimal changes to the codebase therefore kept things as singular.

Reply:
I think it's fine as "multiple LSP servers" is sort of an implementation detail.

Reply:
https://github.com/zed-industries/zed/pull/29359/commits/34f26837da8fa9770d3ac7de3907182d373303a8